main.go

```diff
@@ -125,19 +155,22 @@ func argsParse(argv []string) (noPublish bool, sourcesArg []string, whiteListArg
 		fmt.Sprintf("%s %s", ProgramName, version), false)
 
 	// Parse argument values as usable types.
-	noPublish = arguments["--no-publish"].(bool)
-	sourcesArg = strings.Split(arguments["--sources"].(string), ",")
-	whiteListArg = arguments["--label-whitelist"].(string)
+	args.noPublish = arguments["--no-publish"].(bool)
+	args.sources = strings.Split(arguments["--sources"].(string), ",")
+	args.labelWhiteList = arguments["--label-whitelist"].(string)
+	args.oneshot = arguments["--oneshot"].(bool)
+	args.sleepInterval, _ = strconv.Atoi(arguments["--sleep-interval"].(string))
```

Comment:
It shouldn't end up in crashloopbackoff because it's a daemon, i.e. it doesn't exit, right(?) When using parseduration it probably makes sense to restrict it to some sane value, e.g. 1 second

Reply:
I misread the code last time. It should not crashloopbackoff here. Thanks.